src/PHPMailer.php

```diff
@@ -1,21 +1,13 @@
 <?php
-/**
- * PHPMailer - PHP email creation and transport class.
- * PHP Version 5.5.
+
+/*
+ * This file is part of PHP CS Fixer.
  *
- * @see       https://github.com/PHPMailer/PHPMailer/ The PHPMailer GitHub project
+ * (c) Fabien Potencier <fabien@symfony.com>
+ *     Dariusz Rumiński <dariusz.ruminski@gmail.com>
  *
- * @author    Marcus Bointon (Synchro/coolbru) <phpmailer@synchromedia.co.uk>
- * @author    Jim Jagielski (jimjag) <jimjag@gmail.com>
- * @author    Andy Prevost (codeworxtech) <codeworxtech@users.sourceforge.net>
- * @author    Brent R. Matzelle (original founder)
- * @copyright 2012 - 2017 Marcus Bointon
- * @copyright 2010 - 2012 Jim Jagielski
- * @copyright 2004 - 2009 Andy Prevost
- * @license   http://www.gnu.org/copyleft/lesser.html GNU Lesser General Public License
- * @note      This program is distributed in the hope that it will be useful - WITHOUT
- * ANY WARRANTY; without even the implied warranty of MERCHANTABILITY or
- * FITNESS FOR A PARTICULAR PURPOSE.
+ * This source file is subject to the MIT license that is bundled
+ * with this source code in the file LICENSE.
```

Comment:
Looks like php-cs-fixer went a bit nuts here. I've found it's quite buggy and unhelpful - and the upstream project seems to actively discourage fixing things.

Reply:
i have installed php-cs-fixer locally, but it "fixes" things in the hole file.
so i have created two new patches
one with the S/MIME issue
and one for the AltBody handling